### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by apancoast** [nathansutton/hospital-price-transparency#4] [General]
Found this dataset through Kaggle and just wanted to say a huge thanks, @nathansutton! I'm a fellow North Carolinian, and I'm excited to dig into it for a portfolio project.

**Comment by nathansutton** [comment]
Hey @apancoast best of luck.

I've helped a number of students transition into data roles in industry.  A couple [extra places](https://medium.com/@nasutton/10-excellent-healthcare-data-sources-531468e3dee1) to look.

Feel free to reach out on [SharpestMinds](https://mentors.sharpestminds.com/nathan-sutton) if you could use help.
